timed/tracking/migrations/0007_migrate_activity_blocks.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+
+def migrate_blocks(apps, schema_editor):
+    Activity = apps.get_model('tracking', 'Activity')
+    for activity in Activity.objects.all():
+        if activity.blocks.count() == 1:
```

Comment:
I have a feeling this will lead to duplication of activities as the activity which has more then one block will stay behind.

I guess the first activity block is always an update and the others should be added.

Reply:
Changed it so that there is an index, so it updates on the first iteration and on more it creates.